ndcube/ndcube.py

```diff
@@ -214,9 +214,13 @@ def world_axis_physical_types(self):
         axes_ctype = []
         for i, axis in enumerate(self.missing_axis):
             if not axis:
-                key = ctype[i]
-                if "-TAN" in key:
-                    key = key[:-4]
+		# Find keys in wcs_ivoa_mapping dict that represent start of CTYPE.
+		keys = list(filter(lambda key: ctype.startswith(key), wcs_ivoa_mapping))
+		# If there are multiple valid keys, raise an error.
+		if len(keys) != 1:
+			raise ValueError("Non-unique CTYPE key.  Please raise an issue at https://github.com/sunpy/ndcube/issues citing the following CTYPE and non-unique keys: CTYPE = {0}; keys = {1}".format(ctype, key))
```

Comment:
While you're fixing the indentation (see comment below), could you also break this error message onto multiple lines to be consistent with the Python style rules?  All lines should be less than 100 characters, including indentation spaces.  The fix is very easy.  Simply replace this line (L221) with the following lines:
```
raise ValueError("Non-unique CTYPE key.  Please raise an issue at "
		 "https://github.com/sunpy/ndcube/issues citing the"
		 "following  CTYPE and non-unique keys: "
		 "CTYPE = {0}; keys = {1}".format(ctype, key))
```
Make sure the subsequent lines start with the same indentation as the first ```"``` in the first line, like above.  Your text editor should do this automatically for you.